dash/testing/plugin.py

```diff
@@ -49,6 +49,12 @@ def pytest_addoption(parser):
         help="set this flag to control percy finalize at CI level",
     )
 
+    dash.addoption(
+        "--pause",
+        action="store_true",
+        help="pause using pdb after opening the test app, so you can interact with it",
```

Comment:
FYI - this is a feature I've been thinking about for a while now - very often I want to run an app from the test suite and live-debug it, I would end up copying it into a temp file, fiddling with the indentation and imports to get it to run standalone... now you can just do eg:
```
pytest -k cbwc001 --pause
```
And you're in a Python debugger - where, after advancing a few times, you can run selenium / `dash_duo` commands - but you also have the app from the test `cbwc001` open in a browser window and can interact with it and debug in JS.

Reply:
Oh! Very nice :shipit: